src/raftstore/store/store.rs

```diff
@@ -222,9 +222,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 debug!("handling snapshot for {}", region_id);
                 let task = SnapTask::new(peer.storage.clone());
                 debug!("task generated");
-                match self.snap_worker.schedule(task) {
-                    Err(e) => error!("failed to schedule snap task {}", e),
-                    Ok(_) => peer.storage.wl().snap_state = SnapState::Generating,
+                peer.storage.wl().snap_state = SnapState::Generating;
```

Comment:
this change would overwrite the previous snap_state, will it cause trouble somewhere else?
...is the snap processing serialization and the previous result has been processed(notified)